[2022-03-21 21:32:51,919] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: EL_dag.extract_task scheduled__2012-02-02T01:00:00+00:00 [queued]>
[2022-03-21 21:32:51,925] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: EL_dag.extract_task scheduled__2012-02-02T01:00:00+00:00 [queued]>
[2022-03-21 21:32:51,925] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-21 21:32:51,926] {taskinstance.py:1244} INFO - Starting attempt 5 of 6
[2022-03-21 21:32:51,926] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-21 21:32:51,934] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): extract_task> on 2012-02-02 01:00:00+00:00
[2022-03-21 21:32:51,937] {standard_task_runner.py:52} INFO - Started process 2009 to run task
[2022-03-21 21:32:51,940] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'EL_dag', 'extract_task', 'scheduled__2012-02-02T01:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/extract_load_dags.py', '--cfg-path', '/tmp/tmp8nu1bn_p', '--error-file', '/tmp/tmp2rh6r_49']
[2022-03-21 21:32:51,941] {standard_task_runner.py:77} INFO - Job 31: Subtask extract_task
[2022-03-21 21:32:52,050] {logging_mixin.py:109} INFO - Running <TaskInstance: EL_dag.extract_task scheduled__2012-02-02T01:00:00+00:00 [running]> on host 7fae4c5ec6da
[2022-03-21 21:32:52,076] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.9/site-packages/airflow/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-03-21 21:32:52,095] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=EL_dag
AIRFLOW_CTX_TASK_ID=extract_task
AIRFLOW_CTX_EXECUTION_DATE=2012-02-02T01:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2012-02-02T01:00:00+00:00
[2022-03-21 21:32:52,095] {logging_mixin.py:109} INFO - 02 2012
[2022-03-21 21:32:54,779] {logging_mixin.py:109} INFO - the response is  200
[2022-03-21 21:32:54,780] {logging_mixin.py:109} INFO - here.........
[2022-03-21 21:32:54,812] {logging_mixin.py:109} INFO -   invoice_line_no                     date  ... sale_liters sale_gallons
0    S03862700023  2012-02-01T00:00:00.000  ...        3.00         0.79
1    S04010800110  2012-02-09T00:00:00.000  ...        0.75         0.20
2    S04275700014  2012-02-27T00:00:00.000  ...       10.50         2.77
3    S04023000039  2012-02-09T00:00:00.000  ...        9.00         2.38
4    S04191400094  2012-02-21T00:00:00.000  ...        0.75         0.20

[5 rows x 24 columns]
[2022-03-21 21:32:54,830] {logging_mixin.py:109} INFO - Data saved..........
[2022-03-21 21:32:54,830] {python.py:175} INFO - Done. Returned value was: None
[2022-03-21 21:32:54,842] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=EL_dag, task_id=extract_task, execution_date=20120202T010000, start_date=20220321T213251, end_date=20220321T213254
[2022-03-21 21:32:54,883] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-21 21:32:54,965] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
